nemo/core/classes/modelPT.py

```diff
@@ -1469,3 +1473,33 @@ def on_train_batch_end(self, outputs, batch: Any, batch_idx: int, unused: int =
                     if batch_idx == self._nsys_profile_end_step and get_rank() in self._nsys_profile_ranks:
                         logging.info("====== End nsys profiling ======")
                         torch.cuda.cudart().cudaProfilerStop()
+
+    def cuda(self, device=None):
```

Comment:
Just adding a note that we need to remove this as soon as PTL 7.2 is out with the fix.